guides/source/development_dependencies_install.md

```diff
@@ -43,215 +43,115 @@ $ git clone https://github.com/rails/rails.git
 $ cd rails
 ```
 
-### Set up and Run the Tests
+### Install Additional Tools and Services
 
-The test suite must pass with any submitted code. No matter whether you are writing a new patch, or evaluating someone else's, you need to be able to run the tests.
+Some Rails tests depend on additional tools that you need to install before running those specific tests.
 
-Install first SQLite3 and its development files for the `sqlite3` gem. On macOS
-users are done with:
+Here's the list of each gems' additional dependencies:
 
-```bash
-$ brew install sqlite3
-```
+* Action Cable depends on Redis
```

Comment:
Might be better to link to each gem's README? For example, Action Cable: https://github.com/rails/rails/tree/master/actioncable#configuration